tests/perfsys.ts

```diff
@@ -20,29 +20,9 @@ module perftest {
     export var getCurrentDirectory = ts.sys.getCurrentDirectory;
     var exit = ts.sys.exit;
 
-    var args = ts.sys.args;

```

Comment:
Not necessary, I thought it would fix the travis failed build. Can't reproduce the error it's showing.